html/components/_tooltip.scss

```diff
@@ -62,16 +59,16 @@
     -webkit-transform: rotateY(0deg) rotate(45deg);
     -ms-transform: rotateY(0deg) rotate(45deg);
     transform: rotateY(0deg) rotate(45deg);
-    content: " ";
+    content: ' ';
```

Comment:
do we use single quotes in CSS?

Reply:
I see a mix of single and double in the code right now. This change was made by one of the linters.